src/security/cert_generators/file/security_file_certgen.cc

```diff
@@ -324,9 +324,11 @@ int main(int argc, char *argv[])
             Ssl::CertificateDb::Check(db_path, max_db_size, fs_block_size);
         }
 
+#if HAVE_LIBSSL_SSL_LOAD_ERROR_STRINGS
```

Comment:
This SSLeay_add_ssl_algorithms() is missing a check for itself. We cannot really guarantee that all possible environments will be lacking both. Better to check for both and wrap separately.

Same for the calls in ssl/support.cc

Reply:
Initialization is done automatically in openssl 1.1, so neither call needs to be made.  Openssl 1.1 has a replacement, which can be used in case of a non-standard initialization, which is not the case in squid.  There should be no need to call one or the other separately, so the extra check is not needed.